docs/vfio-user.rst

```diff
@@ -538,11 +547,8 @@ This command message is sent by the client to the server to inform it of the
 memory regions the server can access. It must be sent before the server can
 perform any DMA to the client. It is normally sent directly after the version
 handshake is completed, but may also occur when memory is added to the client,
-or if the client uses a vIOMMU. If the client does not expect the server to
-perform DMA then it does not need to send to the server VFIO_USER_DMA_MAP
-commands. If the server does not need to perform DMA then it can ignore such
```

Comment:
Why remove this? I think it's helpful, maybe make it a note?

Reply:
It just seemed to confuse Stefan at least. Both parts seem kind of obvious, and real implementations don't seem that likely to do either.

Reply:
fair enough